tests/Tests/x__pipelines/x__data-science-pipelines-api.robot

```diff
@@ -0,0 +1,46 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline
+Resource            ../../Resources/RHOSi.resource
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Library             DateTime
+Library             ../../../libs/OpenShiftPipelineAPI.py
+Suite Setup         RHOSi Setup
+Suite Teardown      RHOSi Teardown
+
+
+*** Variables ***
+${KUBEFLOW_OPERATOR_YAML}=                     tests/Resources/Files/kubeflow-operator.yaml
+${DATA_SCIENCE_PIPELINES_APPLICATION_YAML}=    tests/Resources/Files/data-science-pipelines.yaml
+
+*** Test Cases ***
+Install Kubeflow Operator
+    [Documentation]    Install and verifies that Kubeflow Operator is installed and working
+    [Tags]    Smoke
+    Oc Apply    kind=KfDef    src=${KUBEFLOW_OPERATOR_YAML}     namespace=redhat-ods-applications
+    ${pod_count}    Run    oc get pods -n redhat-ods-applications | grep data-science-pipelines-operator-controller-manager | wc -l
+    Should Be True    ${pod_count} == 3    All the pods were created
+
+
+Install DataSciencePipelinesApplication CRD
+    [Documentation]    Install and verifies that DataSciencePipelinesApplication CRD is installed and working
+    [Tags]    Smoke
+    Run    oc new-project ods-ci-pipeline
+    Oc Apply    kind=DataSciencePipelinesApplication    src=${DATA_SCIENCE_PIPELINES_APPLICATION_YAML}     namespace=ods-ci-pipeline
+    ${generation_value}    Run    oc get datasciencepipelinesapplications -o json | jq '.items[0].metadata.generation'
+    Should Be True    ${generation_value} == 1    DataSciencePipelinesApplication created
+
+
+Test and end-to-end pipeline workflow via API
```

Comment:
Tests should not be dependent on the execution of the previous one and should be repeatable (deletion of resources can be done in Test Teardown/Suite teardown)